client/src/components/forms/MoodForm.js

```diff
@@ -1,7 +1,274 @@
+        />
+        <FormGroup>
+          <FormControl className={classes.formControl}>
+            <InputLabel id="sleep-attribute">Sleep</InputLabel>
+            <Select
+              labelId="sleep-attribute"
+              id="sleep-attribute"
+              value={sleep}
+              onChange={handleSleepChange}
+            >
+              <MenuItem value={1}>1 hr</MenuItem>
+              <MenuItem value={2}>2 hrs</MenuItem>
+              <MenuItem value={3}>3 hrs</MenuItem>
+              <MenuItem value={4}>4 hrs</MenuItem>
+              <MenuItem value={5}>5 hrs</MenuItem>
+              <MenuItem value={6}>6 hrs</MenuItem>
+              <MenuItem value={7}>7 hrs</MenuItem>
+              <MenuItem value={8}>8 hrs</MenuItem>
+              <MenuItem value={9}>9 hrs</MenuItem>
+              <MenuItem value={10}>10 hrs</MenuItem>
+              <MenuItem value={11}>11 hrs</MenuItem>
+              <MenuItem value={12}>12 hrs</MenuItem>
+              <MenuItem value={13}>13 hrs</MenuItem>
+              <MenuItem value={14}>14 hrs</MenuItem>
+              <MenuItem value={15}>15 hrs</MenuItem>
+              <MenuItem value={16}>16 hrs</MenuItem>
+              <MenuItem value={17}>17 hrs</MenuItem>
+              <MenuItem value={18}>18 hrs</MenuItem>
+              <MenuItem value={19}>19 hrs</MenuItem>
+              <MenuItem value={20}>20 hrs</MenuItem>
+              <MenuItem value={21}>21 hrs</MenuItem>
+              <MenuItem value={22}>22 hrs</MenuItem>
+              <MenuItem value={23}>23 hrs</MenuItem>
+              <MenuItem value={24}> 24 hrs</MenuItem>
+            </Select>
+          </FormControl>
+          <FormControl className={classes.formControl}>
+            <InputLabel id="med-attribute">Medication</InputLabel>
+            <Select
+              labelId="med-attribute"
+              id="med-attribute"
+              value={med}
+              onChange={handleMedChange}
+            >
+              <MenuItem value={"zoloft"}>Zoloft (sertraline) </MenuItem>
+              <MenuItem value={"celexa"}>Celexa (citalopram)</MenuItem>
+              <MenuItem value={"prozac"}>Prozac (fluoxetine)</MenuItem>
+              <MenuItem value={"desyrel"}>Desyrel (trazodone)</MenuItem>
+              <MenuItem value={"lexapro"}>Lexapro (escitalopram) </MenuItem>
+              <MenuItem value={"cymbalta"}>Cymbalta (duloxetine)</MenuItem>
+            </Select>
+          </FormControl>
+          <FormControl className={classes.formControl}>
+            <InputLabel id="exercise-attribute">Excerise</InputLabel>
+            <Select
+              labelId="exercise-attribute"
+              id="exercise-attribute"
+              value={exercise}
+              onChange={handleExerciseChange}
+            >
+              <MenuItem value={false}>No</MenuItem>
+              <MenuItem value={true}>Yes</MenuItem>
+            </Select>
+          </FormControl>
+          {excercizeTimeInput(exercise)}
```

Comment:
You could convert this Yes/No and Choose Amount pieces into their own component, it will clean up this component a bit too and have less duplication of code.